gammapy/data/hdu_index_table.py

```diff
@@ -17,7 +17,7 @@ class HDUIndexTable(Table):
     See :ref:`gadf:hdu-index`.
     """
 
-    VALID_HDU_TYPE = ["events", "gti", "aeff", "edisp", "psf", "bkg"]
+    VALID_HDU_TYPE = ["events", "gti", "aeff", "edisp", "psf", "bkg", "rad_max"]
```

Comment:
Technically, we don't have different format specifications for the HDU index table, so I suppose it is fine to add `rad_max` even though it does not appear in the v0.2 format specification. 
At some point we will need to introduce different conventions for the valid HDU types. But this will be for later PRs.

Reply:
As this is a backwards compatible addition, I don't see why this should be excluded. Also I would argue that the omission of RAD MAX for the index table hdu in 0.2 of the spec was a bug in the spec, as all other HDU types were listed.